# IN-BIOS5000 / IN-BIOS9000 Autumn 2022 

## Genome Sequencing Technologies, Assembly, Variant Calling and Statistical Genomics

The information below is preliminary and will be updated.


## Introduction

Here you will find information about the autumn 2022 edition of the credited courses [**IN-BIOS5000**](http://www.uio.no/studier/emner/matnat/ifi/IN-BIOS5000) (MSc level) and [**IN-BIOS9000**](http://www.uio.no/studier/emner/matnat/ifi/IN-BIOS9000) (PhD level), on Genome Sequencing Technologies, Assembly, Variant Calling and Statistical Genomics.
The course is offered by the Department of Informatics and the Department of Biosciences at the University of Oslo (UiO) in collaboration with contributors from Oslo University Hospital and the Norwegian Veterinary Institute.
It is partially funded by the [**NORBIS, the national research school in bioinformatics, biostatistics and systems biology**](https://norbis.w.uib.no/genome-sequencing-technologies-assembly-variant-calling-and-statistical-genomics-2/).

This course consists of two weeks of lectures and practicals, a home project with an oral exam presentation, and a digital written exam.


## Audience

The course is open for master and PhD students at UiO, as well as to PhD students registered through the NORBIS national research school. To earn credits for the course, PhD students from other institutions than UiO must obtain the status as a [visiting PhD candidate](https://www.mn.uio.no/english/research/phd/educational-component/visiting-phd.html) at the UiO Faculty of Mathematics and Natural Sciences. Applications may be submitted starting 15 June 2022 and must be submitted by 1 August 2022. A maximum of 30 students in total will be admitted to the course.


## Course content

The course provides knowledge of sequencing technologies and hands-on experience with the analysis of data from several sequencing platforms and for various applications. The two main applications that will be covered are de novo genome assembly and variant calling (SNPs and structural variants), while other aspects like control of quantity and quality of data will also be included. Furthermore, the course will show how to use statistical genomics to analyze features of annotated genomes.


## Learning outcome

After completing this course, you will:

* understand the differences, benefits and drawbacks of the most current high throughput sequencing (HTS) technologies, and be able to decide which platform to use in what way for the different applications of HTS
* be able to evaluate data quality and quantity as well as perform bioinformatics analysis, both on the command-line and through web-based frameworks, with data tailored towards applications like genome assembly and variant calling
* know the algorithms and statistical methods involved in sequence alignment, mapping, assembly and variant calling
* understand how to apply statistical methods to analyse relationships between annotated features on genomic tracks
* be able to report on bioinformatics analysis in such a way that the methods used and steps taken are transparent, thus enhancing reproducibility
* be aware of, and know to deal with, the ethical and data-sensitivity issues surrounding sequencing data derived from human subjects
* be able to critically evaluate, validate and judge the results of bioinformatics analysis of HTS experiments in terms of underlying assumptions, reliability, sensitivity and specificity, and evaluate their value for answering biological questions


## Location

The course will be held at the UiO campus, in seminar room 2269 *Python*, on floor 2 in the northern end of
[Ole-Johan Dahl's building](https://www.uio.no/english/about/getting-around/areas/gaustad/ga06/), Gaustadalléen 23B, Oslo.


## Schedule

Lectures and practical sessions will be held Monday to Friday, 17th to 28th of October, 2022 (weeks 42 and 43). Course days are usually from 09.15 to 17.00. There will be a lunch break each day from 12.00 to 13.15.

In the table below, teachers are indicated with their initials and listed in the following section. Teaching assistant Ghadi al Hajj will help during the exercises.

| Time          | Session (Teacher) |
| ------------- | ----------------- |
| **Week 42**
| **Mon 17 Oct**
| 09.15 - 09.30 | Intro to NORBIS (RB)
| 09.30 - 12.00 | Fundamentals of Molecular Biology (RS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 14.00 | Sequencing and applications (TR)
| 14.15 - 15.00 | Illumina sequencing technology (AS)
| 15.15 - 16.00 | PacBio sequencing technology (ATK)
| **Tue 18 Oct**
| 09.15 - 10.00 | Oxford Nanopore sequencing technology (Zoom) (TH)
| 10.15 - 12.00 | Library preparation and experimental design (AS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 15.00 | Basic file formats (AS)
| 15.15 - 17.00 | Access to computational resources (TR)
| **Wed 19 Oct**
| 09.15 - 12.00 | Pre-processing of HTS data (AS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 17.00 | Small RNA transcriptomics (TBR)
| **Thu 20 Oct**
| 09.15 - 12.00 | *De novo* genome assembly (KL)
| 12.00 - 13.15 | Lunch break
| 13.15 - 17.00 | *De novo* genome assembly, continued (KL)
| 20.00 - 22.00 | Dinner at restaurant in Oslo
| **Fri 21 Oct**
| 09.15 - 12.00 | *De novo* genome assembly, continued (KL)
| 12.00 - 13.15 | Lunch break
| 13.15 - 17.00 | *De novo* genome assembly, continued (KL)
| &nbsp; | |
| **Week 43**
| **Mon 24 Oct**
| 09.15 - 12.00 | Variant calling in general (BS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 15.00 | Variant calling in general, continued (BS)
| 15.15 - 17.00 | Human germline variant calling (YS)
| **Tue 25 Oct**
| 09.15 - 12.00 | Human germline variant calling, continued (YS)
| 12.00 - 13.15 | Lunch break
| 13.15 - 16.00 | Somatic variant calling and interpretation in the context of cancer (SN)
| **Wed 26 Oct**
| 09.15 - 12.00 | Statistical genomics (IG)
| 12.00 - 13.15 | Lunch break
| 13.15 - 17.00 | Statistical genomics, continued (IG)
| **Thu 27 Oct**
| 09.15 - 12.00 | Statistical genomics, continued (IG)
| 12.00 - 13.15 | Lunch break
| 13.15 - 16.00 | Statistical genomics, continued (IG)
| 16.15 - 17.00 | Pizza on campus
| **Fri 28 Oct**
| 09.15 - 12.00 | Machine learning in computational biology (MP)
| 12.00 - 13.15 | Lunch break
| 13.15 - 16.00 | Machine learning in computational biology, continued (MP)
| 16.15 - 16.30 | Exam info and course evaluation (TR)
| &nbsp; | |
| **Week 48**
| **Thu 1 Dec**
| 09.15 - 15.00 | Oral presentation and exam
| 17.00 - 19.00 | Written exam
| **Fri 2 Dec**
| 09.15 - 15.00 | Oral presentation and exam


## Teachers

| Initials | Name                    | Affiliation                                                         |
| -------- | ----------------------- | ------------------------------------------------------------------- |
| AS       | Arvind Sundaram         | Norwegian Sequencing Centre, Oslo University Hospital
| ATK      | Ave Tooming-Klunderud   | CEES, Dept. of Biosciences, University of Oslo
| BS       | Bastiaan Star           | CEES, Dept. of Biosciences, University of Oslo
| GAH      | Ghadi al Hajj           | Centre for Bioinformatics, Dept. of Informatics, University of Oslo
| IG       | Ivar Grytten            | Centre for Bioinformatics, Dept. of Informatics, University of Oslo
| KL       | Karin Lagesen           | Section for Epidemiology, Norwegian Veterinary Institute
| MP       | Milena Pavlovic         | Centre for Bioinformatics, Dept. of Informatics, University of Oslo
| RB       | Ragna Breines           | NORBIS, Dept. of Informatics, University of Bergen
| RS       | Rolf Skotheim           | Centre for Bioinformatics, Dept. of Informatics, University of Oslo
| SN       | Sigve Nakken            | Oslo University Hospital
| TH       | Thomas Haverkamp        | Section for Epidemiology, Norwegian Veterinary Institute
| TR       | Torbjørn Rognes         | Centre for Bioinformatics, Dept. of Informatics, University of Oslo
| TBR      | Trine B. Rounge         | Centre for Bioinformatics, Dept. of Pharmacy, University of Oslo
| YS       | Ying Sheng              | Dept. of Medical Genetics, Oslo University Hospital


## Contact information

* Torbjørn Rognes (Course coordinator), email torognes@ifi.uio.no


## Meals

Coffee and tea will be freely available each morning in the seminar room.

For lunch, we recommend the canteen in the other end of the building, [Ole-Johan spiseri](https://www.sio.no/en/food-and-beverage/our-canteens-and-coffeeshops/informatikk-kafeen), which serves a nice selection of warm and cold dishes.

We plan to organize a free dinner for all course participants in a restaurant in Oslo on October 20th.

We also plan to order pizza on October 27th. Details will follow.


## Requirements

* We recommend that participants should have a basic understanding of molecular biology as provided by an introductory course in bioinformatics, molecular biology, or genetics.

* No formal background in computer science is required, however, students must have a basic understanding of the unix shell. Students should take an introductory unix course beforehand if they do not have these skills.


## Computing facilities

All students must have a laptop with either Windows (version 10 or more recent), macOS, Linux or other Unix operating system. You need to be able to access the Eduroam wifi network. If you do not have a UiO or Eduroam account, a temporary Wifi account may be created for you.

We advise you to bring an external mouse, and do not rely on touchpad/trackpad only.

The course will use computational resources on the Educloud Fox system at UiO. [Access to Educloud](Educloud/index.md) will be provided for all course participants. A [list of the software packages](software.md) that will be used is available.

If you are struggling with anything of the above, you must contact the course coordinator (see contact details on this page) as soon as possible, and at least one week before the start of the course.


## Curriculum and suggested reading
 
The curriculum consists of a set of scientific articles. The exam will contain questions on the material presented in these papers on the material covered during the course. Specific details of the curriculum, and a list of further suggested papers, can be found [here](Curriculum/index.md).


## Exam

The exam will be a combination of the following two parts:

 * an *individual* home project combined with an oral exam presentation of the home project followed by questioning / discussion on December 1st and 2nd. Students will present their work to 2 teachers, followed by some questioning (approx 20 minutes in total for each student). This home exam will be handed out to all participants after the lectures. Time slots will be assigned randomly.
 * a 2 hour written digital exam at Silurveien 2 on December 1st 2022 from 17.00 to 19.00.

The exams are graded on a pass/fail scale.

Written exams from previous years:

* [2018](https://github.com/arvindsundaram/IN-BIOSx000/blob/2018/misc/written_exam.pdf)
* [2016](https://github.com/lexnederbragt/INF-BIOx121/blob/2017/misc/2016.pdf) (exam only)
* [2015](https://github.com/lexnederbragt/INF-BIOx121/tree/2015/exam) (with information on how the grading was done)
* [2014](https://wiki.uio.no/projects/clsi/images/f/f2/Examinfbio2014.pdf) (exam only)


## Lecture slides

[Lecture slides](Lectures/index.md)<br/>

This page is updated after each lecture/Module.


## Teaching materials

List of further suggested papers, can be found [here](Curriculum/index.md).


## Previous editions of the course

Note: For previous editions of the IN-BIOS5000 / IN-BIOS9000 course please see the [2018](https://in-biosx000.readthedocs.io/en/2018/) or [2020](https://in-biosx000.readthedocs.io/en/2020/) websites. The course was not held in 2019 and 2021. For previous editions of the INF-BIO9121 and INF-BIO5121 course, please see the [2013](https://wiki.uio.no/projects/clsi/index.php/INF-BIOX120_H13), [2014](https://wiki.uio.no/projects/clsi/index.php/INF-BIOX121_H14), [2015](https://inf-biox121.readthedocs.io/en/2015/), [2016](https://inf-biox121.readthedocs.io/en/2016/) or [2017](https://inf-biox121.readthedocs.io/en/2017/) website. The 2012 version did not have a dedicated available website.
