# Software that will be used in the course

Several software packages for general sequence handling, alignment, genome assembly, assembly quality assessment, and variant calling will be used during the course. Below is a list of the the packages that will be used. They will be made available on Fox. They are all already available on Saga.

Package | Version | Kind | Documentation | Source code repository
--------|---------|------|---------------|-----------------------
assembly-stats | 0.1.4 | Python library | [https://pypi.org/project/assembly-stats](https://pypi.org/project/assembly-stats) | [https://github.com/MikeTrizna/assembly_stats](https://github.com/MikeTrizna/assembly_stats)
bedtools | 2.30.0 | Binary | [https://bedtools.readthedocs.io/en/latest](https://bedtools.readthedocs.io/en/latest) | [https://github.com/arq5x/bedtools2](https://github.com/arq5x/bedtools2)
BWA | 0.7.17 | Binary | [https://github.com/lh3/bwa](https://github.com/lh3/bwa) | [https://github.com/lh3/bwa](https://github.com/lh3/bwa)
bwa-mem2 | 2.2.1 | Binary | [https://github.com/bwa-mem2/bwa-mem2](https://github.com/bwa-mem2/bwa-mem2) | [https://github.com/bwa-mem2/bwa-mem2](https://github.com/bwa-mem2/bwa-mem2)
FastQC | 0.11.9 | Java | [https://www.bioinformatics.babraham.ac.uk/projects/fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc) | [https://github.com/s-andrews/FastQC](https://github.com/s-andrews/FastQC)
FASTX-Toolkit | 0.0.14 | Binary | [http://hannonlab.cshl.edu/fastx_toolkit](http://hannonlab.cshl.edu/fastx_toolkit) | [https://github.com/agordon/fastx_toolkit](https://github.com/agordon/fastx_toolkit)
GATK | 4.2.6.1 | Java | [https://gatk.broadinstitute.org/hc/en-us](https://gatk.broadinstitute.org/hc/en-us) | [https://github.com/broadinstitute/gatk](https://github.com/broadinstitute/gatk)
MultiQC | 1.12 | Binary | [https://multiqc.info](https://multiqc.info) | [https://github.com/ewels/MultiQC](https://github.com/ewels/MultiQC)
Pilon | 1.24 | Binary | [http://software.broadinstitute.org/software/pilon](http://software.broadinstitute.org/software/pilon) | [https://github.com/broadinstitute/pilon](https://github.com/broadinstitute/pilon)
QUAST | 5.2.0 | Binary | [http://cab.spbu.ru/software/quast](http://cab.spbu.ru/software/quast) | [https://github.com/ablab/quast](https://github.com/ablab/quast)
SAMtools | 1.16 | Binary | [https://www.htslib.org](https://www.htslib.org) | [https://github.com/samtools/samtools](https://github.com/samtools/samtools)
SPAdes | 3.15.5 | Binary | [http://cab.spbu.ru/software/spades](http://cab.spbu.ru/software/spades) | [https://github.com/ablab/spades](https://github.com/ablab/spades)
Trimmomatic | 0.39 | Java | [http://www.usadellab.org/cms/?page=trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic) | [https://github.com/usadellab/Trimmomatic](https://github.com/usadellab/Trimmomatic)
Unicycler | 0.5.0 | Binary | [https://github.com/rrwick/Unicycler](https://github.com/rrwick/Unicycler) | [https://github.com/rrwick/Unicycler](https://github.com/rrwick/Unicycler)
Velvet | 1.2.10 | Binary | [https://github.com/dzerbino/velvet](https://github.com/dzerbino/velvet) | [https://github.com/dzerbino/velvet](https://github.com/dzerbino/velvet)

To be updated...
