# Curriculum and supporting material 2022


##### Molecular Biology

Refer to Wikipedia for basic information.

[NHGRI/NIH's Talking Glossary of Genomic and Genetic Terms](https://www.genome.gov/genetics-glossary)


##### High throughput sequencing (HTS)

[The sequence of sequencers: The history of sequencing DNA](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_history.pdf), Heather JM _et. al._, Genomics 2016

[High-Throughput Sequencing Technologies](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_technology.pdf), Reuter JA _et. al._, Molecular Cell 2015

[Sequencing depth and coverage: key considerations in genomic analyses](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/HTS_Coverage_Depth.pdf), Sims D _et. al._, Nature Reviews 2014

[Single molecule real-time (SMRT) sequencing comes of age: applications and utilities for medical diagnostics](https://doi.org/10.1093/nar/gky066), Simon Ardui, Adam Ameur, Joris R Vermeesch, Matthew S Hestand (2018) Nucleic Acids Research, 46, 5, 2159–2168.

[Nanopore sequencing technology, bioinformatics and applications](https://doi.org/10.1038/s41587-021-01108-x), Wang, Y., Zhao, Y., Bollas, A. et al. (2021) Nat Biotechnol 39, 1348–1365.


##### _De novo_ genome assembly

[A field guide to whole-genome sequencing, assembly and annotation](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/DNA_1.pdf), Ekblom and Wolf, Evolutionary Applications 2014

[The Theory and Practice of Genome Sequence Assembly](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/DNA_2.pdf), Simpson and Pop, Annual Review of Genomics and Human Genetics 2015

[The present and future of de novo whole-genome assembly](https://doi.org/10.1093/bib/bbw096), Jang-il Sohn, Jin-Wu Nam (2018) Briefings in Bioinformatics, 19 (1), 23-40.

[A survey on de novo assembly methods for single-molecular sequencing](https://dx.doi.org/10.1007/s40484-020-0214-5), Ying Chen, Chuan-Le Xiao (2020) Quant. Biol., 8, 3, 203-215.


##### Variant calling

[Genotype and SNP calling from next-generation sequencing data](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/VC_1.pdf) Nielsen et al., Nature Reviews Genetics 2011. This is a review paper that should be easily understandable once the student has taken the variant calling module. 

[Exome sequencing identifies the cause of a mendelian disorder](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/VC_2.pdf) Ng et al., Nature Genetics 2010. A paper reporting the first resolution of a mendelian disorder using exome capture.

[Variant calling: Considerations, practices, and developments](https://doi.org/10.1002/humu.24311), Zverinova, S., &  Guryev, V. (2022) Human Mutation,  43,  976-985.

[Best practices for variant calling in clinical sequencing](https://doi.org/10.1186/s13073-020-00791-w), Koboldt, D.C. (2020) Genome Med 12, 91.

[Getz, et al.](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/SVC_1.pdf) "Advances in understanding cancer genomes through second-generation sequencing". Nature Reviews Genetics 2010

[A review of somatic single nucleotide variant calling algorithms for next-generation sequencing data](https://doi.org/10.1016/j.csbj.2018.01.003) Chang Xu (2018), Computational and Structural Biotechnology Journal, 16, 15-24.


##### Statistical genomics

[Colocalization analyses of genomic elements: approaches, recommendations and challenges, Kanduri et. al.](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/SG_1.pdf), Bioinformatics 2018.

[Simovski, Boris, et al. "GSuite HyperBrowser: integrative analysis of dataset collections across the genome and epigenome."](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/SG_2.pdf)

[Quinlan, Aaron R., and Ira M. Hall. "BEDTools: a flexible suite of utilities for comparing genomic features."](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/SG_3.pdf) Bioinformatics 26.6 (2010): 841-842.

[Optional reading for those interested: Salvatore, Stefania, et al.](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/SG_4.pdf) "Beware the Jaccard: the choice of similarity measure is important and non-trivial in genomic colocalisation analysis." Briefings in bioinformatics 21.5 (2020): 1523-1530


##### Machine Learning

[Wainberg, et al. Deep learning in biomedicine](https://github.com/arvindsundaram/IN-BIOSx000/raw/2020/Curriculum/ML.pdf), Nature Biotechnology 2018

[DOME: recommendations for supervised machine learning validation in biology](https://doi.org/10.1038/s41592-021-01205-4), Walsh, I., Fishman, D., Garcia-Gasulla, D. et al. (2021) Nat Methods 18, 1122–1127.

[Navigating the pitfalls of applying machine learning in genomics](https://doi.org/10.1038/s41576-021-00434-9), Whalen, S., Schreiber, J., Noble, W.S. et al. (2022) Nat Rev Genet 23, 169–181.
